src/lib/icon/icon.component.ts

```diff
@@ -134,9 +134,7 @@ export class FaIconComponent implements OnChanges {
       const message = 'Global icon library is deprecated. ' +
         'Consult https://github.com/FortAwesome/angular-fontawesome/blob/master/UPGRADING.md ' +
         'for the migration instructions.';
-      if (this.config.globalLibrary === 'unset') {
-        console.error('FontAwesome: ' + message);
-      } else if (!this.config.globalLibrary) {
+      if (!this.config.globalLibrary || this.config.globalLibrary === 'unset') {
```

Comment:
Please revert this change. 

`'unset'` value should behave the same as before: print warning. This option may be used by the consumers with the large codebases to opt-out of the hard error, while they update their applications.

Instead [change the default value](https://github.com/FortAwesome/angular-fontawesome/blob/master/src/lib/config.ts#L39) from `'unset'` to `false` in the `FaConfig` as described in the doc comment on that property.

Reply:
Understood. Reverted.